ocpp/charge_point.py

```diff
@@ -213,8 +216,12 @@ async def _handle_call(self, msg):
             handler = handlers['_after_action']
             # Create task to avoid blocking when making a call inside the
             # after handler
-            asyncio.ensure_future(
-                asyncio.coroutine(handler)(**snake_case_payload))
+            response = handler(**snake_case_payload)
+            if inspect.isawaitable(response):
+                response = await response
+                asyncio.ensure_future(
+                    response)
+            response
```

Comment:
Not sure what the function of this line is. Can you elaborate?

Reply:
it is not needed it, removed here: https://github.com/mobilityhouse/ocpp/pull/109/commits/3adfd2515625d3bde449f57c5bca0a03f58493dd